### PR TITLE
Update read.html

### LINKED ISSUE #5904: Uploaded files and activity stream history
**CKAN version**
2.9.1

**Describe the bug**
When viewing an uploaded resource via the activity stream history, the url and downloaded filename match the prior version, but the file contents is the most recently uploaded file.  There is a red message a the top that could lead the user to believe that they would be getting the older version of the resource.

**Steps to reproduce**
Create a dataset with an uploaded file resource
Manage the resource and upload a new file in its place
Using the activity stream browse back to the old version of the dataset
Try to download the resource from the old dataset
The contents of the file will be the newer file

**Expected behavior**
I don't know, maybe if the file has been replaced the URL listing should removed and a message put in its place

An option to retain old versions would be awesome, but that is a bigger ask

### PR DESCRIPTION
**Pull request by Gauravp-NEC**
Fixes #5904 

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by Gauravp-NEC** [comment]
@amercader , Please review this PR.

**Comment by Gauravp-NEC** [comment]
@amercader , Please review this PR. Thankyou.

**Comment by Gauravp-NEC** [comment]
@amercader ,Gentle reminder. Please review this PR. If it looks good please merge this PR. Thankyou.

File: ckan/templates/package/read.html [hunk 1]
```diff
@@ -13,8 +13,9 @@
       {% if is_activity_archive %}
         <div class="alert alert-danger">
           {% trans url=h.url_for(pkg.type ~ '.read', id=pkg.id) %}
-          You're currently viewing an old version of this dataset. To see the
-          current version, click <a href="{{ url }}">here</a>.
+          You're currently viewing an old version of this dataset.
+          Resource files may not match the old version of the data.
+          To see the current version, click <a href="{{ url }}">here</a>.
```
Comment (amercader) [inline ckan/templates/package/read.html:18]:
```suggestion
          You're currently viewing an old version of this dataset.
          Data files may not match the old version of the metadata.
          <a href="{{ url }}">View the current version</a>.
```

File: ckan/templates/package/resource_read.html [hunk 2]
```diff
@@ -75,8 +75,9 @@
             {% if is_activity_archive %}
               <div id="activity-archive-notice" class="alert alert-danger">
                 {% trans url=h.url_for(pkg.type ~ '.read', id=pkg.id) %}
-                You're currently viewing an old version of this dataset. To see the
-                current version, click <a href="{{ url }}">here</a>.
+                You're currently viewing an old version of this dataset.
+                Resource files may not match the old version of the data.
+                To see the current version, click <a href="{{ url }}">here</a>.
```
Comment (amercader) [inline ckan/templates/package/resource_read.html:80]:
```suggestion
          You're currently viewing an old version of this dataset.
          Data files may not match the old version of the metadata.
          <a href="{{ url }}">View the current version</a>.
```
```

**Review by amercader — CHANGES_REQUESTED** [review]

**Comment by Gauravp-NEC** [comment]
@amercader , based on your suggestion I have done the rework. Now this PR is ready to be merged. Thankyou :)

**Pull request merged**